recipes/guidata/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.7.6" %}
+
+package:
+  name: guidata
+  version: {{ version }}
+
+source:
+  url: https://github.com/PierreRaybaut/guidata/archive/v{{ version }}.tar.gz
+  sha256: 2ffc8e6dd27a57c6ec0e260986368beeddbd2fa5588f5200614b307b27a39e37
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - guidata-tests-py2 = cguidata.tests:run  # [py2]
+    - guidata-tests-py3 = cguidata.tests:run  # [py3]
```

Comment:
Oops, that should be `py2k` and `py3k` respectively.

Reply:
Sure! I'm going to change, thanks! :)